utils/google.js

```diff
@@ -238,6 +238,35 @@ module.exports = {
       }) );
   },
 
+  /**
+   * @param {string} projectID
+   * @returns {Object[]} a list of service accounts
+   */
+  async listServiceAccounts(projectID) {
+    return new Promise(resolve => googleApp.authorize(googleApp.cloudManagerConfig, (authClient) => {
+      const cloudResourceManager = google.iam('v1');
+      const request = {
+          name: `projects/${projectID}`,
+        auth: authClient,
+      };
+      cloudResourceManager.projects.serviceAccounts.list(request, (err, res) => {
+        if (err) {
+          if (err instanceof Error) {
+            resolve(err);
+          } else {
+            resolve(Error(err));
+          }
+          return;
+        }
+        if (res && res.data) {
+          resolve(res.data['accounts']);
+        } else {
+          resolve(Error(`Unexpected get service account result: ${JSON.stringify(res)}`));
```

Comment:
Add a comment or change the Error message to indicate that we're ignoring errors.  Or better return an empty list - something like that.  Or pass the error back to the caller with a `reject`.

Reply:
Comment added.